packages/ebsr/configure/src/main.jsx

```diff
@@ -26,12 +26,42 @@ export class Main extends React.Component {
     onConfigurationChanged: PropTypes.func
   };
 
+  onModelChanged = (model, key) => {
+    const { onModelChanged } = this.props;
+
+    switch (key) {
+      case 'partB.choiceMode': {
+        let value = model.partB.choiceMode;
+
+        if (value === 'radio') {
+          let correctFound = false;
+
+          model.partB.choices = model.partB.choices.map(c => {
+            if (correctFound) {
+              c.correct = false;
+              return c;
+            }
+
+            if (c.correct) {
+              correctFound = true;
+            }
+            return c;
```

Comment:
I think you can use only one "if" here:

if (c.correct && correctFound) {
c.correct = false;
correctFound = true;
}
return c;

Please correct me if this doesn't what you're expecting.

Reply:
'correctFound' becomes true when first correct answer is found. then all 'c.correct' are made false